seed/models/scenarios.py

```diff
@@ -91,11 +91,17 @@ def copy_initial_meters(self, source_scenario_id):
         from seed.models.meters import Meter
 
         source_scenario = Scenario.objects.get(pk=source_scenario_id)
+        try:
+            property_ = PropertyView.objects.get(state=self.property_state).property
```

Comment:
Potentially no change here - just curious when a -State we care about doesn't have a corresponding -View? Is this a mid-merge type of situation?

Reply:
Per conversation, we'll replace the exception with a informative comment. But we do want to keep it here in case there are cases we didn't consider.